### PR TITLE
KATI-119_MessageListServer

### PR DESCRIPTION
**Pull request by toannguyen71195**

### CODE REVIEW

File: com.kms.cura.entity/src/com/kms/cura/entity/MessageEntity.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package com.kms.cura.entity;
+
+import java.sql.Timestamp;
+import com.kms.cura.entity.user.DoctorUserEntity;
+import com.kms.cura.entity.user.UserEntity;
+
+public class MessageEntity extends Entity {
+	private UserEntity sender, receiver;
+	private Timestamp timeSent;
```
Comment (hieumai) [inline com.kms.cura.entity/src/com/kms/cura/entity/MessageEntity.java:9]: Should not create field for time with type java.sql.Timestamp. Use Date instead.

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @vtnlinh95  please start verifying on this.

**Comment by toannguyen71195** [comment]
Guide:
[Guide to verify.docx](https://github.com/hieumai/cura/files/377941/Guide.to.verify.docx)

**Comment by hieumai** [comment]
OK from me. @toannguyen71195  please rebase so I can merge

**Pull request merged**